transform/livereloadinject/livereloadinject.go

```diff
@@ -72,23 +57,9 @@ func New(baseURL url.URL) transform.Transformer {
 		c := make([]byte, len(b))
 		copy(c, b)
```

Comment:
I didn't write those lines, but isn't this pointless? From my (limited) understanding of Go:
- `make` wasn't passed a capacity, so future calls to `append(c, ...)` will need to reallocate
- `copy` was given the entirety of `b` even though we're about to append to `c[:idx]`, overwriting most of it

My intuition would be to write this lower in the function instead:

    c := make([]byte, idx, len(b)+len(script))
    copy(c, b)

Or alternatively:

    c := make([]byte, 0, len(b)+len(script))
    c = append(c, b[:idx]...)

Reply:
You are right, this would be better:

```go
c := make([]byte, idx, len(b)+len(script))
copy(c, b)
```